doc/adrs/010-glossary.adoc

```diff
@@ -0,0 +1,40 @@
+= ADR-010 - Adopt a Shared Vocabulary
+
+== Context
+
+Given our long history with Sirius Desktop and the way it used modeling and EMF-specific terms, we tend to reuse these (at least in our discussions).
+For Sirius Web, we do not want to be limited to EMF, or even a modeling/MDE point of view, se we need to use different terms.
+
+We need to decide on a common vocabulary, both for the user-visible elements (e.g. do we talk about "Documents", or "Models"? is it "Modeler" or "Studio", or do both exist but represent slightly different things?) and also more internally on the technical terms.
+
```

Comment:
We should add a small definition for each roles and concepts.

Reply:
yes, this is just an *very* early doc to initiate the discussion